### PR TITLE
Replace tabs with spaces

### PR DESCRIPTION
**Pull request by timowest**

### CODE REVIEW

**Comment by Shredder121** [comment]
Did you check other files for potential tabs?

**Comment by timowest** [comment]
I extended this now.

File: querydsl-apt/src/main/hibernate.xml [hunk 1]
```diff
@@ -7,13 +7,12 @@
   </formats>
   <includeBaseDirectory>false</includeBaseDirectory>
   <fileSets>
+    <fileSet>      <directory>src/apt/hibernate</directory>
```
Comment (Shredder121) [inline querydsl-apt/src/main/hibernate.xml]: What happened here?

**Comment by Shredder121** [comment]
As for the contents of this PR, it's not a feature that has to be reviewed so merging it early is fine with me.

**Comment by johnktims** [comment]
It looks like you found all the tabs so I'll merge this.

**Pull request merged**